descqa/readiness_test.py

```diff
@@ -126,11 +126,12 @@ def __init__(self, **kwargs):
         if not all(d.get('quantity') for d in self.uniqueness_to_check):
             raise ValueError('yaml file error: `quantity` must exist for each item in `uniqueness_to_check`')
 
-        if not all(d.get('quantity') for d in self.catalog_filters):
-            raise ValueError('yaml file error: `quantity` must exist for each item in `catalog_filters`')
+        if len(self.catalog_filters) > 0:
```

Comment:
code style suggestion
```suggestion
        if self.catalog_filters:
```